neurom/io/utils.py

```diff
@@ -153,7 +193,7 @@ def load_neurons(neurons,
     for f in files:
         try:
             pop.append(neuron_loader(f))
-        except NeuroMError as e:
+        except (NeuroMError, MorphioError) as e:
             if isinstance(e, ignored_exceptions):
                 L.info('Ignoring exception "%s" for file %s',
                        e, f.name)
```

Comment:
I cannot comment on the raise but don t you want to :
```
raise NeuroMError(e) from e
```
?
So you only return neurom errors and no morphio error

Reply:
Why hiding MorphIO errors? By Mike's request I removed encapsulation of MorphIO errors by NeuroM errors. If continue following his request, this raise should stay as is.

Reply:
I usually don t like having an error from another package popping up. The `from e` will raise a neurom error but still show you the MorphIO error.
This is to avoid : 
```
try: 
     neurons = load_neurons(...)
except (MorphIOError, NeuromError): # with multiple error sources here
    blabla
```

Reply:
ok, makes sense, done